certbot/client.py

```diff
@@ -53,9 +54,15 @@ def determine_user_agent(config):
     """
 
     if config.user_agent is None:
-        ua = "CertbotACMEClient/{0} ({1}) Authenticator/{2} Installer/{3}"
-        ua = ua.format(certbot.__version__, util.get_os_info_ua(),
-                       config.authenticator, config.installer)
+        if cli.cli_command in ["certbot", "letsencrypt", "certbot-auto", "letsencrypt-auto"]:
+            auto = cli.cli_command
+        else:
+            auto = "other"
```

Comment:
Why throw out the data of what `cli.cli_command` is here, rather than in post-processing? Better to have the data then throw it out later, no?